packages/oui-password/src/rule/password-rule.controller.js

```diff
@@ -31,8 +31,8 @@ export default class {
         );
 
         this.$scope.$watch(
-            () => this.password.model,
-            (value) => this.setValidity(value)
+            () => `${this.password.model}${this.password.validationTrigger}`,
```

Comment:
I think we could keep it like it was, with a simpler attribute ;)

Reply:
This has be undone